packages/amplify-provider-awscloudformation/src/configuration-manager.js

```diff
@@ -617,6 +712,8 @@ function getConfigLevel(context) {
         // configLevel is 'general' only when it's explicitly set so
         if (envConfigInfo.configLevel === 'general') {
           configLevel = 'general';
```

Comment:
could be `envConfigInfo.configLevel`

Reply:
Leaving as is to maintain current functionality.